theseus_gui/src/routes/accounts/index.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
```

Comment:
in the future, don't worry about creating UI pages, since the whole UI module is being rewritten in Vue

Reply:
Sounds good, I'll remove this file from version control